gemsfx-demo/nbactions.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please delete netbeans config file from project.

Reply:
removed it